### DISCUSSION TITLE
Site looks different after v4->v5 upgrade. Why? How to fix?

### DISCUSSION THREAD

**Discussion by dgh328** [ant-design/ant-design#41854] [General]
Hi, I am attempting to upgrade my site from Ant Design v4 to v5. To my surprise, the  new look is quite a bit off:

- Current Site, in v4: https://zarobky.pl/
- Same site, transitioned to v5: https://stage.zarobky.pl/

I have read through https://ant.design/docs/react/migration-v5 and I understand that primary color is different and edges have been rounded to 6px. However, there are many more changes here, such as:
- Different font
- Different line spacing
- Different icon sizes
- Font color
- More...

I have not made any changes other than upgrade to v5 (and update some backward-incompatible components):
```
yarn upgrade antd --latest
```

What could be the cause of the discrepancy and what is the best approach to make the v5 look more like the original?

**Comment by abenhamdine** [comment]
Did you have a less file configured in your v4 app ? Have you migrated it ? 
https://ant.design/docs/react/migration-v5#less-migration
